release-team/role-handbooks/test-infra/README.md

```diff
@@ -178,6 +174,11 @@ It is important to monitor Tide after config changes are made for code slush, fr
 Until the CNCF infra migration is complete, a member of Google's gke-engprod team will need to monitor Tide logs.
 However, most of Tide's behavior can be monitored without access to the cluster. The [Tide dashboard](https://prow.k8s.io/tide) and [Velodrome monitoring dashboard](http://velodrome.k8s.io/dashboard/db/monitoring?orgId=1) provide insight into what Tide is currently doing, how much load it is handling, and how it is performing.
 
+### Test-Infra 'Code Freeze'
+
+The stability of our test infra is critical to getting reliable testing signals throughout the release cycle, but the signal is most important at the end of the release cycle during code slush and freeze. While the `kubernetes/test-infra` repo does not enforce additional merge restrictions related to the release cycle, we do try to limit the changes that are merged. Specifically, during slush and freeze, changes to test-infra should be limited to important fixes and work that doesn't impact critical infrastructure.
```

Comment:
change release-related job/infra needs extra care, but changes for other jobs are still fine?

Reply:
unless there is a clear list of what changes are  release-related job/infra related, I would err on the side of caution and restrict any big changes to test-infra in general. In the past 2 cycle we are trying to shorten code freeze to just 2 weeks. So hopefully this isnt too much of a disruption.